applications/API/filters.py

```
@@ -0,0 +1,47 @@
+    This filter enables filtering insults by NSFW status, category, and status fields.
+    """
+
+    """
+    Provides filtering options for user-submitted Insult objects in the API.
+
+    This filter enables filtering insults by NSFW status, category, and status fields.
+    """
+
+    class Meta:
+        model = Insult
+        fields = {"nsfw": ["exact"], "category": ["exact"], "status": ["exact"]}
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)